compiler/circle-opselector-test/README.md

```diff
@@ -0,0 +1,52 @@
+# Circle-OpSelector-test
+
+## Usage
```

Comment:
Please copy and paste this test tool's help message.

For example, refer to https://github.com/Samsung/ONE/tree/master/tools/nnpackage_tool/gen_golden#readme or https://github.com/Samsung/ONE/blob/master/tools/nnpackage_tool/model2nnpkg/README.md.